notebooks/community/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -261,7 +261,7 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    PROJECT_ID = ""  # @param {type:"string"}

use template version for setting PROJECT_ID
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>